package.json

```diff
@@ -3,6 +3,9 @@
   "version": "1.1.1",
   "description": "rnpm plugin that links native dependencies to your project",
   "main": "index.js",
+  "scripts": {
+    "test": "eslint ./ && mocha ./test --recursive"
```

Comment:
when referenced from inside npm scripts, `node_modules/bin` is added to PATH so these are actually called from inside node_modules (local stuff).